sample/EasyTables/CSharp/QueueToCollector/function.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "disabled": true,
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "samples-workitems"
```

Comment:
The existing QueueTrigger sample already binds to this queue, which means that if your samples are also enabled they'd be competing for messages and we'd get inconsistent results.